tests/ember_debug/object-inspector-test.js

```diff
@@ -1005,4 +1013,59 @@ module('Ember Debug - Object Inspector', function(hooks) {
       ]);
     });
   }
+
+  // @glimmer/component 1.0 doesn't seem to support 3.4, even though it has the manager API
```

Comment:
Does this mean there will be errors with people using less than 3.8? We have to support 3.4+ currently.

Reply:
No, this comment is saying that it isn't actually possible to use `@glimmer/component` on Ember < 3.8 at all (nothing to do with Ember Inspector). 

If we can, we should fix that issue, but it's unrelated to the specific changes here.